install.sh

```diff
@@ -56,8 +81,10 @@ Include = /etc/pacman.d/mirrorlist
 pacman --noconfirm -Sy
 pacman --noconfirm -S \
 	lightdm \
+	lightdm-gtk-greeter \
 	accountsservice \
 	xorg-server \
+	xorg-apps \
```

Comment:
what is this needed for?

Reply:
Its a part of X.org server that contains utilities which could potentially fix some game issues, due to lack of utilities that games require.